modules/log6x-logging-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,39 @@
+[id="log6x-logging-compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Some features or components in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features or components are not intended for production use.
+
+In the following tables, features or components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+NA:: Not Applicable
+
+[NOTE]
+====
+To know about compatible {product-title} versions for {logging-uc} releases, see the link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product life cycle page].
```

Comment:
```suggestion
To know about compatible {product-title} versions for {logging-uc} releases, see the link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product Life Cycles].
```

Reply:
Link now matches title of document. Not sure if/don't think you need to include "page".